mono/mini/TestDriver.cs

```diff
@@ -51,6 +52,9 @@ public class TestDriver {
 				} else if (args [j] == "--exclude") {
 					exclude [args [j + 1]] = args [j + 1];
 					j += 2;
+				} else if (args [j] == "--exclude-test") {
+					exclude_test.Add (args [j + 1]);
```

Comment:
This is unwieldy in several cases, say if there are multiple test cases crashing and I'm picking them off one by one (multiple --exclude-test options are accepted) I might want to switch sets rapidly, adding a category also requires me to change my working copy which under some circumstances would be awkward.

Overall our current interface is weird because we have a --run-only which takes a test name but no equivalent for exclude.